matlab/Ice/context/client.m

```diff
@@ -0,0 +1,59 @@
+% Copyright (c) ZeroC, Inc.
+
+function client(args)
+    % We put the code generated by the Slice compiler in the 'generated' subdirectory.
+    addpath('generated');
+
+    % Load the Ice library if it is not already loaded.
+    if ~libisloaded('ice')
+        loadlibrary('ice', @iceproto);
+    end
+
+    % The Slice module VisitorCenter maps to a MATLAB package with the same name.
+    import VisitorCenter.*
+
+    if nargin == 0
+        args = {};
```

Comment:
Should the readme mention this args?

Reply:
I don't think we need to.

Reply:
Also, it doesn't work, so we should fix it!